styles.css

```diff
@@ -0,0 +1,995 @@
+    font-size: 1.5rem !important;
+    letter-spacing: 0.2rem;
+    font-weight: 900;
+  }
+
+  .logo:hover {
+    color: #424141;
+  }
+
+  .link-menu {
+    color: #8b8787 !important;
+    cursor: pointer;
+    margin: auto !important;
+    border: #fff 0.25rem solid;
+  }
+
+  .link-menu:hover {
+    color: #ec5242 !important;
+  }
+
+  .about-link,
+  .home-link {
+    color: #ec5242 !important;
+  }
+
+  .about-link:hover,
+  .home-link:hover {
+    color: #fff !important;
+    background-color: #ec5242;
+  }
+
+  .link-campaign {
+    color: #ec5242 !important;
+    border: #ec5242 0.25rem solid;
+  }
+
+  .link-campaign:hover {
+    color: #fff !important;
+    background-color: #ec5242;
+    margin: auto !important;
+  }
+
+  .header {
+    border-bottom: #d3d3d3 1px solid;
+    box-shadow: 0 0 0.5rem;
+  }
+
+  .nav-item {
+    margin-right: 2rem;
+  }
+
+  .title-about {
+    margin: 4rem 0 1rem;
+    font-size: 2.5rem;
+    line-height: 3rem;
+  }
+
+  .bar-about-main {
+    display: block;
+    width: 2rem;
+    height: 0.06rem;
+    margin: auto;
+    background: #ec5242;
+  }
+
+  .description-about-mobile {
+    display: none;
+  }
+
+  .description-about-desktop {
+    display: block;
+    margin: 0;
+  }
+
+  .about {
+    width: 90%;
+    margin: 2rem 5% 3rem 5%;
+    padding: 1.5rem;
+    color: #424141;
+  }
+
+  .contact,
+  .email {
+    font-size: 1rem;
+  }
+
+  .logo-title-h3 {
+    display: none;
+  }
+
+  .logo-title-h3-desktop {
+    display: block;
+    padding: 2rem 0 0.25rem;
+    text-align: center;
+    color: #424141;
+    font-size: 1.75rem;
+    font-weight: 400;
+  }
+
+  .about-ministries {
+    margin: 1rem 20%;
+    width: 60%;
+    text-align: center;
+    color: #424141;
+    font-size: 1rem;
+  }
+
+  .about-logo-p-desktop {
+    display: block;
+  }
+
+  .about-logo-p {
+    display: none;
+  }
+
+  .about-img {
+    width: 30%;
+    margin: auto;
+  }
+
+  .conference-images {
+    flex-direction: row;
+    justify-content: center;
+    width: 100%;
+  }
+
+  .conference1,
+  .conference2 {
+    width: 30%;
+  }
+
+  .partner {
+    display: none;
+  }
+
+  .copyright {
+    background-color: #424141;
+    height: 10rem;
+    padding: 3rem;
+  }
+
+  .logo-footer {
+    font-size: 1.5rem !important;
+    flex: 0.6;
+  }
+
+  .logo-footer:hover {
+    color: #d3d3d3;
+  }
+
+  .rights-declaration {
+    display: none;
+  }
+
+  .rights-declaration-desktop {
+    display: block;
+    color: #d3d3d3;
+    font-size: 0.8rem;
+  }
+
+  /* home page */
+  .greet {
+    font-size: 2.5rem;
+  }
+
+  .title {
+    font-size: 3.25rem;
+    line-height: 4rem;
+    font-weight: 900;
+  }
+
+  .description {
+    width: 62%;
+  }
+
+  .main-activities {
+    display: flex;
+    justify-content: center;
+    gap: 0.125rem;
+    flex-wrap: wrap;
+  }
+
+  .t-main {
+    width: 10% !important;
+    margin: 0;
+    opacity: 0.7;
+  }
+
+  .t-main:hover {
+    opacity: 1;
+    transition: all 0.3s ease-in-out;
+  }
+
```

Comment:
- [ ] Kindly add the hover effect to each card as a shown a [Behance design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) 🙏🏼 which is a white border. You can use `border: 1px solid white;` 💪